JoomGalleryfriends/JG4-dev
# Configuration manager

## What is this pull request for

Basic functionality of Configuration Manager realized. It is now possible to use all the buttons and functionalities of the Configuration Manager.
It is now possible to reset/export/import configuartion sets.

## How to test this  pull request

- Click all the buttons, links and try everything out that is possible with the GUI inside the Configuration Manager.

It is not ment to test if the configuration settings are correctly applied within the component. It is about to test the functionality of the config manager itself like
- Creating, deleting, copying Configuration sets
- Reset, Export, Import Configuration sets
- Changing settings and see if the changes were saved in the configuration set (deleting, adding, changing settings)
- List view functionality like filtering, searching, ordering, status change, ...
